.gitignore

```diff
@@ -12,14 +12,16 @@
 *.rbc
 *.seed
 .dat*
+.DS_Store
 .grunt
+.idea
 .lock-wscript
 .node_repl_history
 .npm
 .nyc_output
 .repl_history
 .rvmrc
-.idea
```

Comment:
You're just playing favorites.

Reply:
ah i honestly don't remember ever doing this haha, i'll revert it

Reply:
yeah, i started alphabetizing this then stopped. i can revert this if needed

Reply:
I was just joking, i misread the diff and thought you deleted .idea in favor of .vscode :)